cards.js

```diff
@@ -17,7 +17,100 @@ function shuffle(array){
 }
 
 function buildDeck(){
-
+  var deck = [];
+  for (var i=0; i<suits.length; i++) {
+    for (var j=0; j<values.length; j++) {
+      var card= {suit: suits[i], value: values[j], rank: j};
+      deck.push(card);
+    }
+  }
+  return deck;
 // Put some code in here!
+}
 
+function highCard(deck) {
+  var player1= deck[0];
+  var player2= deck[1];
+  console.log("Player 1: " + player1.value)
+  console.log("Player 2: " + player2.value)
+  if (player1.rank > player2.rank) {
+    console.log("Player 1 wins high card!")
+  }else if (player1.rank === player2.rank) {
+    console.log("High card is a tie!")
+  }else {
+    console.log("Player 2 wins high card!")
+  }
 }
+
+
+function war(deck1, deck2) {
+  var player1= deck1;
+  var player2= deck2;
+  var turn = 0
+  while (player1 != "" && player2 != "" ) {
```

Comment:
Do the quotes here and elsewhere indicate a placeholder for unfinished code? `player1 != ""`